DigiMeSDK/Core/Classes/Entities/DMEOAuthToken.h

```diff
@@ -0,0 +1,29 @@
+//
+//  DMEOAuthToken.h
+//  DigiMeSDK
+//
+//  Created on 12/12/2019.
+//  Copyright © 2019 digi.me Limited. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DMEOAuthToken : NSObject
+
+// access token - gives access to protected resources via the digi.me Public API, without requiring users to disclose their digi.me credentials.
+@property (nonatomic, strong, nullable) NSString *accessToken;
```

Comment:
All the `NSString` properties here should use `copy` instead of `strong`. This prevents a scenario where a mutable string is set as property on the Auth object in and then its value is changed (thus also changing the Auth object's value)
e.g.
```
NSMutableString *token = @"aaa";
DMEOAuthObject * authObject = [DMEOAuthObject new];
authObject.accessToken = token; // accessToken is now "aaa"
[token appendString:@"bbb"]; // Both token and authObject.accessToken are "aaabbb"

```